docs/source/development.rst

```diff
@@ -0,0 +1,46 @@
+Development
+===========
+
+In order to work on ``datajudge``, you can create development conda environment as follows:
+
+::
+
+    git clone https://github.com/Quantco/datajudge
+    cd datajudge
+    mamba env create
+    conda activate datajudge
+    pip install --no-build-isolation --disable-pip-version-check -e .
+
+Unit tests can be run by executing
+
+::
+
+   pytest tests/unit
+
+Integration tests are run against a specific backend at a time. As of now, we provide helper
+scripts to spin up either a postgres or mssql backend.
```

Comment:
The database engines are personal nouns, as such I guess they should have some capitalization, unless they're explicitly named all lowercase like in `pytest`.

Reply:
[bae3481](https://github.com/Quantco/datajudge/pull/17/commits/bae34813d6495dc5adbf9e06c358f8b8055d01b2)